source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -623,23 +612,22 @@ MedcoMD engineers write a function to create a new patient record:
 
       .. code-block:: javascript
 
-         function insertPatient(collection, name, ssn, bloodType, medicalRecords,  policyNumber, provider) {
-             collection.insertOne({
-                name: name,
-                ssn: ssn,
-                bloodType: bloodType,
-                medicalRecords: medicalRecords,
-                insurance: {
-                    policyNumber: policyNumber,
-                    provider: provider
-                }
-             })
-             .then((writeResult) => {
-                 console.log("writeResult:", writeResult);
-             })
-             .catch((writeError) => {
-                 console.log("writeError occurred:", writeError);
-             })
+         async function insertPatient(collection, name, ssn, bloodType, medicalRecords, policyNumber, provider) {
+           try {
+             const writeResult = await collection.insertOne({
+               name: name,
+               ssn: ssn,
+               bloodType: bloodType,
+               medicalRecords: medicalRecords,
+               insurance: {
+                 policyNumber: policyNumber,
+                 provider: provider
+               }
+             });
+             console.log('writeResult:', writeResult);
```

Comment:
do you want the `writeResult` to be returned to the user at all? For example:

```js
const writeResult = await insertPatient(coll, 'dan', '123-456-7890', ...);
```

if so, it should be returned after this `console.log`

Reply:
Whoops, thanks for spotting this. It does not look like we use the result in any other snippet. Will remove.